packages/expo-image/ios/EXImage.m

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020-present 650 Industries. All rights reserved.
+
+#import <expo-image/EXImage.h>
+
+@implementation EXImage
+
+RCT_EXPORT_MODULE()
+
+- (UIView *)view
+{
+    // TODO: Implement some actually useful functionality
+    UILabel * label = [[UILabel alloc] init];
+    [label setTextColor:[UIColor redColor]];
+    [label setText: @"*****"];
+    [label sizeToFit];
+    UIView * wrapper = [[UIView alloc] init];
+    [wrapper addSubview:label];
+    return wrapper;
+}
+
+@end
```

Comment:
I've checked a bunch of `.m` and `.h` files in the repo and indentation is set to `2` spaces. Probably we should enforce consistency somehow as I strongly dislike the idea of `use whatever you like`.

Reply:
> Probably we should enforce consistency somehow as I strongly dislike the idea of use whatever you like.

Me too, wouldn't have committed 4-space indentation if it was a file created by me (it was generated).